CHANGELOG.md

```diff
@@ -1,31 +1,31 @@
 # CHANGELOG
 
-## [Unreleased]
-
+## v3.2.1
 ### Breaking changes
-- [\#628](https://github.com/Manta-Network/Manta/pull/628) Improve RPC performance, add `max_receivers` and `max_senders` fields in the RPC request.
+- [Dolphin] [\#628](https://github.com/Manta-Network/Manta/pull/628) Improve RPC performance, add `max_receivers` and `max_senders` fields in the RPC request.
 
 ### Features
-- [\#576](https://github.com/Manta-Network/Manta/pull/576) Unfilter xtokens.transfer_multicurrencies and bump MaxAssetsForTransfer to 2.
+- [\#576](https://github.com/Manta-Network/Manta/pull/576) Unfilter `xtokens.transfer_multicurrencies` and bump MaxAssetsForTransfer to 2.
```

Comment:
Not sure why this change occur in both sections `Breaking changes` and `Improvements`.

Reply:
Breaking changes and improvements would be okay.
Breaking is a kind of spotlight section with changes that need extra attention and any kind of change bugfix/imprvement/new feature can break something existing

what shouldnt be the case is that 576 is both a feature and and improvement.

Reply:
Fixed it